### DISCUSSION TITLE
播客相关工具推荐

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [opensource-f2f/episode#229] [General]
## https://podcast.adobe.com/enhance
 
Features:
* [Enhance speech by removing all background noise and echo](https://podcast.adobe.com/enhance)
* [Unlock quality sound from your microphone](https://podcast.adobe.com/miccheck)

**Comment by MikeyWei** [comment]
搭车推荐一篇《[低成本播客音质提升指南](https://mp.weixin.qq.com/s/51DUF_hNCwb3Q39JPYboIQ)》。
另外，关于播客音频文件输出的参数，之前发过一个倡议，因为中间设想有自动转码就 close 掉了，不妨看一个市场调查报告中的播客音频文件数据（码率和增益等）：https://podcast-standard.org/audio/?interval=year

**Reply by wjsvec** [reply]
get! 学习一下~
